trident/absorption_spectrum/absorption_spectrum.py

```diff
@@ -897,18 +903,20 @@ def _add_lines_to_spectrum(self, field_data, use_peculiar_velocity,
                              'increasing the bin size.') % my_vbins.size)
 
                     # the virtual bins and their corresponding opacities
-                    if (line['wavelength'].d == 2.1e9): 
+                    if (line['label'] == '21 cm'): 
```

Comment:
Similar to my comment above, it's somewhat redundant to be able to switch the deposition method and change the behavior for one specific line. Because of the huge range of wavelengths between the hyperfine transition and the electron energy level transitions, I don't think anyone should be making a spectrum that includes both the 21 cm and something from the Lyman series. I'd rather see a workflow where the user specifies that they want only the 21 cm line and the delta deposition method manually. This would remove the need to have separate function calls here, which is going to be more difficult to maintain and sets a precedent for adding even more.

Reply:
Agree with Britton.